internal/state/player.go

```diff
@@ -0,0 +1,14 @@
+package state
+
+type Player struct {
+    location Location
```

Comment:
Do we also need something here to represent the player's controllable actions?

Reply:
yes, will add in a another PR